iothub/device/tests/Authentication/DeviceAuthenticationWithTokenRefreshTests.cs

```diff
@@ -154,7 +154,7 @@ public async Task DeviceAuthenticationWithSakRefresh_SharedAccessKeyConnectionSt
             Assert.IsNotNull(iotHubConnectionCredentials.SasTokenRefresher);
             Assert.IsInstanceOfType(iotHubConnectionCredentials.SasTokenRefresher, typeof(DeviceAuthenticationWithSakRefresh));
 
-            var auth = (IAuthorizationProvider)iotHubConnectionCredentials;
+            var auth = iotHubConnectionCredentials;
```

Comment:
Can we do away with the alias and just use `iotHubConnectionCredentials` directly in line 160 below?